editor/src/components/canvas/canvas-strategies/post-action-options/post-action-paste.ts

```diff
@@ -233,6 +249,11 @@ function pasteChoiceCommon(
         },
       },
     }),
+    wildcardPatch('on-complete', {
+      trueUpGroupsForElementAfterDomWalkerRuns: {
+        $set: groupTrueUpPaths,
+      },
+    }),
```

Comment:
this shall either be a `groupTrueUpPaths.map(p => queueGroupTrueUp(p))`, or we should change `queueGroupTrueUp` to accept an array of ElementPath

Reply:
oh! nice catch, I wanted to be consistent with the rest of the file but it's much better. Went with the former option ;) https://github.com/concrete-utopia/utopia/pull/3946/commits/933d363588fa0b3e33692c9f5c1c7df2c251fd09